local.css

```diff
@@ -431,3 +431,9 @@ table.whitespace caption {
   background-color: #04AA6D;
   color: white;
 }
+
+td.exampleChar {
+  font-size: 36pt;
```

Comment:
Can we use a rem or, if necessary, an em unit rather than pt?  If you really want to fix the size, perhaps use px rather than pt.

Reply:
Done. Made it `300%`